src/Query/Condition.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Drupal\entity\Query;
+
+/**
+ * Represents a single access condition.
+ */
+final class Condition {
+
+  /**
+   * The supported operators.
+   *
+   * @var array
+   */
+  protected static $supportedOperators = [
+    '=', '<>', '<', '<=', '>', '>=', 'BETWEEN', 'NOT BETWEEN',
+    'IN', 'NOT IN', 'IS NULL', 'IS NOT NULL',
```

Comment:
What's the reason this doesn't list `STARTS_WITH`, `CONTAINS` or `ENDS_WITH`?

Reply:
We used only the operators that can be directly passed to the matching SQL condition, because of pure laziness. No reason not to extend it in a followup.